wrangler-docs/functions/json.md

```diff
@@ -0,0 +1,207 @@
+# JSON Functions
+
+These are json functions that can be useful in transforming your json data.
+
+ - `json-string` represents the string version of json.
+ - `json-element` represents either the
+the json object or json array.
+ - `json-object` and `json-array` represent different collections.
+ - `json-null` represents a null element of json.
+
+## Example data
+
+Upload to the workspace `json` an input record such as:
+```
+{
+    "name": {
+        "fname": "Joltie",
+        "lname": "Root",
+        "mname": null
+    },
+    "coordinates": [
+        12.56,
+        45.789
+    ],
+    "numbers": [
+        1,
+        2.1,
+        3,
+        null,
+        4,
+        5,
+        6,
+        null
+    ],
+    "moves": [
+        { "a": 1, "b": "X", "c": 2.8},
+        { "a": 2, "b": "Y", "c": 232342.8},
+        { "a": 3, "b": "Z", "c": null},
+        { "a": 4, "b": "U"}
+    ],
+    "integer": 1,
+    "double": 2.8,
+    "float": 45.6,
+    "aliases": [
+        "root",
+        "joltie",
+        "bunny",
+        null
+    ]
+}
+```
+
+Below is an example malformed json described as `malformed_json` that is missing a comma.
+
+```
+    {
+      a : 1,
+      b : 2.0,
+      c : test
+      d : true
+    }
+```
+
+## Parse
+Parses a json into `json-element`. If there are any issues in parsing, the function returns
+`json-null`
+
+### Namespace
+`json`
+
+### Input
+`json-string`
```

Comment:
the convention here looks different than in other PRs. Please ensure they are consistent